src/Resources/app/storefront/src/js/filter-category-select.plugin.js

```diff
@@ -161,84 +299,91 @@ export default class FilterCategorySelectPlugin extends FilterBasePlugin {
     }
 
     /**
-     * @param ids
+     * Selects the given list of categories.
+     *
+     * @param {String[]} selectedCategories
      * @private
      */
-    _setCurrentCategoryAsSelected(ids) {
-        const selectedCategory = ids.pop();
-
-        // Selected category
-        const checkboxEl = DomAccess.querySelector(this.el, `[id = "${selectedCategory}"]`, false);
-        if (checkboxEl) {
-            this.enableOption(checkboxEl);
-            checkboxEl.disabled = false;
-            checkboxEl.checked = true;
-            this.selection.push(checkboxEl.value);
-        }
+    _selectCategories(selectedCategories) {
+        Iterator.iterate(selectedCategories, (selectedCategory) => {
+            const checkbox = DomAccess.querySelector(this.el, `[id = "${selectedCategory}"]`, false);
+
+            if (checkbox) {
+                this.enableOption(checkbox);
+                checkbox.disabled = false;
+                checkbox.checked = true;
```

Comment:
Enabling in this context means, making it available to click. Also checking it would result in all options being checked by default. This method is pre-defined by Shopware.